tests/pytests/functional/modules/state/requisites/test_listen.py

```diff
@@ -1,8 +1,6 @@
 import pytest
 
-pytestmark = [
-    pytest.mark.windows_whitelisted,
-]
+pytestmark = [pytest.mark.windows_whitelisted, pytest.mark.core_test]
```

Comment:
```suggestion
pytestmark = [pytest.mark.windows_whitelisted, pytest.mark.core_test,]
```
Add the `,` comma above and let black format it when you try to commit.
The same for all `pytestmark` which are list and are on a single line.